opm/simulators/wells/BlackoilWellModel_impl.hpp

```diff
@@ -200,7 +200,7 @@ namespace Opm {
 
             // Wells are active if they are active wells on at least
             // one process.
-            wells_active_ = localWellsActive() ? 1 : 0;
+            wells_active_ = this->well_container_.size() > 0 ? 1 : 0;
```

Comment:
```C++
wells_active_ = !this->well_container_.empty()
```

Reply:
Thanks. The comments can also be updated, but I will wait for the jenkins and benchmark finishes.